src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml

```diff
@@ -72,10 +58,12 @@ sylius_grid:
             list: '@SyliusAdmin/shared/grid/action/list.html.twig'
             show: '@SyliusAdmin/shared/grid/action/show.html.twig'
             update: '@SyliusAdmin/shared/grid/action/update.html.twig'
+            # Resource specific actions
             create_payment_method: "@SyliusAdmin/payment_method/grid/action/create.html.twig"
             create_product_attribute: "@SyliusAdmin/product_attribute/grid/action/create.html.twig"
+            delete_catalog_promotion: "@SyliusAdmin/catalog_promotion/grid/action/delete.html.twig"
             generate_variants: "@SyliusAdmin/product/grid/action/generate_variants.html.twig"
-            ship_with_tracking_code: "@SyliusAdmin/shipment/grid/action/ship_with_tracking_code.html.twig"
+            ship_with_tracking_code: '@SyliusAdmin/shipment/grid/action/ship_with_tracking_code.html.twig'
```

Comment:
Why `'` 🤔? Everywhere else we use `"`

Reply:
copy-paste from the previous one 💃 
Besides from what I see, we're mostly using `'` in this config